### PR TITLE
use ct_expand

### PR DESCRIPTION
**Pull request by fenollp**
Fixes https://github.com/certifi/erlang-certifi/issues/23
Haven't had time to play with Travis yet, I expect that the +deterministic flag will fail for older OTP versions thus needing some rebar.config magic.
Also, @benoitc do you know how is cacerts.pem generated / how frequently? I need this for the Travis Cron.

### CODE REVIEW

**Comment by benoitc** [comment]
Thanks for the patch! I'm not sure for the cront, a new release is generally done every 6 months or less but that's pretty irregular. Will check anyway. For the `+deterministc` flag did you figure when it was added to Erlang ?

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,14 @@ before_install: set -e
 after_script: set +e
 install: curl -#fSLo ~/rebar3 https://s3.amazonaws.com/rebar3/rebar3 && chmod +x ~/rebar3
 after_install: ~/rebar3 version
-script: ~/rebar3 eunit
+script:
```
Comment (benoitc) [inline .travis.yml]: what about having it as a pre script in rebar ? Or maybe using a makefile so we can also fetch it easily by hand

**Comment by fenollp** [comment]
Now by setting up a monthly cron (should be enough) in travis we'd know if the cacerts.pem need any updates, although manual. Making this project really self-sustaining / publishing is more work than I can get into for now.

**Comment by benoitc** [comment]
@fenollp let me test it this morning, I will release it asap :)

**Comment by fenollp** [comment]
WFM ;)

**Comment by benoitc** [comment]
Thanks!, 2.3.0 is on hex

**Pull request merged**